setup.cfg

```diff
@@ -1,2 +1,5 @@
 [aliases]
 test = pytest
+
+[pep8]
+max-line-length = 90
```

Comment:
@akrherz any opinions about this? (80 vs 90 vs 2,204)?

Reply:
not a hill I am willing to die on :)